### ISSUE TITLE
Magento Cloud Docker incompatibility

### ISSUE THREAD

**Issue by shiftedreality** [magento/composer-root-update-plugin#21]
The plugin is not compatible with the latest version of MCD

**Steps to reproduce**

TBA

```
"magento/composer-root-update-plugin: 1.1.0" for the Web Setup Wizard...
Reading /app/var/vendor/magento/composer-root-update-plugin/composer.json
Loading config file /app/var/vendor/magento/composer-root-update-plugin/composer.json
No Web Setup Wizard update needed for magento/composer-root-update-plugin; version 1.1.0 is already in /app/var.
```

**Root cause analysis**

- It looks like the plugin exits with non-zero code which later results in broken deployment

**Comment by slitviachenko** [comment]
You could get this error during Magento CE installation on Magento Cloud Docker because [WebSetupWizardPluginInstaller](https://github.com/magento/composer-root-update-plugin/blob/1.1.0/src/Magento/ComposerRootUpdatePlugin/Setup/WebSetupWizardPluginInstaller.php) isn't skipped. The possible fix is https://github.com/magento/composer-root-update-plugin/pull/22

**Comment by pdohogne-magento** [comment]
Fix released in version 1.1.1
